port/unix/omrsysinfo.c

```diff
@@ -2004,6 +2013,74 @@ omrsysinfo_get_OS_version(struct OMRPortLibrary *portLibrary)
 					rc = sysctlbyname(KERN_OSPRODUCTVERSION, buffer, &resultSize, NULL, 0);
 				}
 				buffer[resultSize] = '\0';
+
+				/* 10.16 is the version returned when in a backwards compatibility context in OSX versions 11+
+				* so we check the plink symlink as that bypasses the compatibility context and returns the
+				* real version info.
+				*/
```

Comment:
nit: asterisks should line up

Reply:
Updated PR to address this formatting issue.